src/modules/sim-constants.js

```diff
@@ -2,213 +2,213 @@
 // Based in part on code from Apache Ripple, https://github.com/apache/incubator-ripple
```

Comment:
yes, i looked at it and we certainly do not need all of them. however, just like you say, one issue at a time. feel free to file a bug for this specific file. there's actually quite a lot of clean up required through out the code, but i think it could be done over time.